ga4gh/frontend.py

```diff
@@ -194,6 +194,7 @@ def configure(configFile=None, baseConfig="ProductionConfig",
     TODO Document this critical function! What does it do? What does
     it assume?
     """
+    requests.packages.urllib3.disable_warnings()
```

Comment:
Right, we need to suppress the warnings that come from the server subprocess, so we can't do it via the test file.

We could wrap this line in a `if app.config['TESTING']:` statement.  Which is better, but then one might deploy to prod and get a bunch of warnings one hasn't seen before... is this acceptable?